pkgs/tools/backup/restic/default.nix

```diff
@@ -26,7 +26,7 @@ buildGoModule rec {
   passthru.tests.restic = nixosTests.restic;
 
   postPatch = ''
-    rm cmd/restic/integration_fuse_test.go
+    rm cmd/restic/cmd_mount_integration_test.go
```

Comment:
Am I missing something, or are we deleting the file we just patched?

Why maintain a patch, when we delete the patched file anyway? Why delete when we have a patch that disable one test instead of removing all?

Reply:
We patch `cmd/restic/cmd_restore_integration_test.go`.

Reply:
So I indeed missed something :D

Anyway, if I remove the `postPatch` locally it still builds for me.

Reply:
Is that on macOS?

Reply:
So the problem solved by the removal of this file is Mac specific?